core/src/main/java/cz/o2/proxima/repository/Repository.java

```diff
@@ -176,37 +169,33 @@ public Repository build() {
    */
   private final Map<String, ValueSerializerFactory> serializersMap = new HashMap<>();
 
-
   /**
    * Map of all entities specified by name.
    * This need not be synchronized because it is only written in constructor
    * and then it is read-only.
    **/
   private final Map<String, EntityDescriptor> entitiesByName = new HashMap<>();
 
-
   /**
    * Map of entities by pattern.
    * This need not be synchronized because it is only written in constructor
    * and then it is read-only.
    **/
-  private final Map<NamePattern, EntityDescriptor> entitiesByPattern;
-
+  private final Map<NamePattern, EntityDescriptor> entitiesByPattern = new HashMap<>();
```

Comment:
Can we remove this whole `entitiesByPattern`? I really don't know why this could be useful.